File: codewars/8kyu/will-you-make-it/dduck-will-you-make-it.js
// https://www.codewars.com/kata/5861d28f124b35723e00005e/train/javascript

const zeroFuel = (distanceToPump, mpg, fuelLeft) => distanceToPump <= (mpg * fuelLeft);


console.log(zeroFuel(50, 25, 2), true);
console.log(zeroFuel(100, 50, 1), false);

File: codewars/8kyu/will-you-make-it/dduck-will-you-make-it.py
# https://www.codewars.com/kata/5861d28f124b35723e00005e/

from unittest import TestCase

zero_fuel = lambda distance, mpg, fuel_left: distance <= mpg * fuel_left

TestCase().assertEqual(zero_fuel(50, 25, 2), True)
TestCase().assertEqual(zero_fuel(100, 50, 1), False)
